pkg/controller/common/common.go

```diff
@@ -0,0 +1,11 @@
+package common
```

Comment:
Let's avoid package, .go files and functions named: "util", "common", "helper".  They lead to _buckets of functions_. I strongly believe that any functionality added should be scoped and grouped by concern.

These functions look like a good fit in the _settings_ package and likely a method on `_Settings`.

Reply:
Sounds good.